src/main/java/net/glowstone/util/lang/LanguageProvider.java

```diff
@@ -12,7 +13,7 @@
     /**
      * Returns the internationalized message according to the effective locale of the player.
      */
-    String getString(String key, GlowPlayer p, Object ... args);
+    String getString(GlowPlayer p, String key, Object ... args);
```

Comment:
That is up to you and how you want to design your system :) I don't think it's a bad idea, as long as it is well documented (possibly in the wiki for future developers, but that can be done in the future).